packages/kbn-text-based-editor/src/helpers.test.ts

```diff
@@ -209,33 +208,6 @@ describe('helpers', function () {
     });
   });
 
-  describe('getInlineEditorText', function () {
```

Comment:
ℹ️ Removed as it is related to the compact mode which is entirely removed in this redesign